java-bootcamp-2018/src/com/globant/unitTest/FactoryProducerUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.globant.unitTest;
+
+import static org.junit.Assert.*;

```

Comment:
Its considered a bad practice to include *
The IDEs provide shortcuts to resolve this automatically, in Eclipse is ctrl + shift + o